Improve Bazel support (#1434)
oneapi-src/oneTBB: Bazel.md

Switch to the folder with the files created earlier and run the binary with `bazel run //:Demo`.

### Bzlmod

> If you are using Bzlmod you can fetch oneTBB using [the Bazel Central Registry](https://registry.bazel.build/).

Reviewer: ```suggestion
If you use Bzlmod, you can fetch oneTBB with the [Bazel Central Registry](https://registry.bazel.build/).
```

> Add to your `MODULE.bazel` file:

Reviewer: ```suggestion
Add the following line to your `MODULE.bazel` file:
```
